components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/v1/impl/KeyManagersApiServiceImpl.java

```diff
@@ -152,9 +164,14 @@ public Response keyManagersPost(KeyManagerDTO body, MessageContext messageContex
                     KeyManagerMappingUtil.toKeyManagerConfigurationDTO(organization, body);
             KeyManagerConfigurationDTO createdKeyManagerConfiguration =
                     apiAdmin.addKeyManagerConfiguration(keyManagerConfigurationDTO);
-            APIUtil.logAuditMessage(APIConstants.AuditLogConstants.KEY_MANAGER,
-                    new Gson().toJson(keyManagerConfigurationDTO),
-                    APIConstants.AuditLogConstants.CREATED, RestApiCommonUtil.getLoggedInUsername());
+            String jsonString = new Gson().toJson(createdKeyManagerConfiguration);
+            try {
+                JSONObject logInfoJson = (JSONObject) new JSONParser().parse(jsonString);
```

Comment:
can't we move this logic inside the APIUtil.logAuditMessage method itself?
Otherwise this is causing lot of repetitive code